tests/sentinel/tests/03-runtime-reconf.tcl

```diff
@@ -197,7 +197,7 @@ test "Set parameters in normal case with bad format" {
 test "Sentinel Set with other error situations" {
 
    # non-existing script
-   assert_error "ERR Notification script seems non existing*" {S 0 SENTINEL SET mymaster notification-script test.txt}
+   assert_error "ERR Notification script seems nonexistent*" {S 0 SENTINEL SET mymaster notification-script test.txt}
```

Comment:
This changes the expected error message that Valkey returns without actually changing the error message in `sentinel.c` itself. I suggest reverting this for now since it's a behavior change.